docs/add-header-and-footer-to-pdf-form.md

```diff
@@ -0,0 +1,137 @@
+# Add Header and Footer to a PDF Form
```

Comment:
Add Page Header and Footer to a PDF form

// The header and footer of a form appears at the begining/ending of a document rather than individual pages.
// I suggest specifically mentioning that it is the header and footer of a _page_ rather than the entire document.